test/Microsoft.AspNetCore.Blazor.Build.Analyzers.Test/Microsoft.AspNetCore.Blazor.Build.Analyzers.Test.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="iomjthbk.12p~" />
```

Comment:
Any idea what this is?

Reply:
Ugh, sorry, will remove. VS adds temporary files with names like this during some operations like renames. I guess it's meant to clean up after itself but didn't do so correctly this time.